config/spryk/spryks/Spryker/Glue/AddGlueBasicStructure.yml

```diff
@@ -0,0 +1,85 @@
+spryk: structure
+description: "Adds a new Glue layer with basic structure."
+mode: both
+
+application: Glue
+
+priority: 1
+
+arguments:
+  module:
+    inherit: true
+
+  organization:
+    inherit: true
+    default: Spryker
+
+  resourceType:
```

Comment:
maybe this should have the module lower cased minus RestApi by default?

Reply:
I thought about it, but we had cases, when module name didn't matched resource name, e.g. `ProductsRestApi` module have two resources `concrete-products` and `abstract-products`.

Reply:
That's what I mean default, overridable of course by input